src/views/gdxfor/gdxfor.jsx

```diff
@@ -204,8 +205,10 @@ class GdxFor extends ExtensionLanding {
                         />
                     </Steps>
                 </ExtensionSection>
-                <ExtensionSection className="faq">
-                    <h2><FormattedMessage id="gdxfor.troubleshootingTitle" /></h2>
+                <ExtensionTroubleshooting
```

Comment:
This is the exception for `deviceName`, the registered trademark is `Go Direct`, which we don't actually use. So this one should keep the `formatMessage` calls.